.github/workflows/build.yml

```diff
@@ -0,0 +1,123 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
this will make all the test run twice when you update a pull request

Reply:
I just copied this part from elsewhere, I don't have much of an opinion about it.

Reply:
I think this is the only important comment. As you can see in this PR, all tests ran twice.